src/platform/user/profile/utilities/index.js

```diff
@@ -8,13 +8,6 @@ import localStorage from '~/platform/utilities/storage/localStorage';
 
 import { ssoKeepAliveSession } from '~/platform/utilities/sso';
 
-import {
```

Comment:
everything deleted from this file was moved to a new `vap-svc/util/index.js` file